src/components/Whiteboard/ElementBehaviors/Resizable/ResizeElement.tsx

```diff
@@ -21,336 +21,275 @@ import {
+      height: boundingRect.height,
+      elements,
+    });
+  }, [activeElements, elements]);
 
   const handleDragStop = useCallback(() => {
-    if (!element || !startDimensions) {
-      return;
-    }
-
-    if (
-      element.type === 'shape' &&
-      (startDimensions.x !== element.position.x ||
-        startDimensions.y !== element.position.y ||
-        startDimensions.width !== element.width ||
-        startDimensions.height !== element.height)
-    ) {
-      slideInstance.updateElement(elementId, {
-        position: { x: element.position.x, y: element.position.y },
-        width: element.width,
-        height: element.height,
-      });
-    } else if (element.type === 'path') {
-      slideInstance.updateElement(elementId, {
-        position: { x: element.position.x, y: element.position.y },
-        points: element.points,
-      });
-    }
+    slideInstance.updateElements(
+      Object.entries(elements).map(([elementId, element]) => {
+        return {
+          elementId,
+          patch: {
+            position: element.position,
+            width: element.type === 'shape' ? element.width : undefined,
+            height: element.type === 'shape' ? element.height : undefined,
+            points: element.type === 'path' ? element.points : undefined,
+          },
+        };
+      }),
+    );
 
-    setStartDimensions(undefined);
-    setElementOverride([{ elementId, elementOverride: undefined }]);
-  }, [element, startDimensions, slideInstance, elementId, setElementOverride]);
+    setResizableProperties(undefined);
+    setElementOverride(createResetElementOverrides(elementIds));
+  }, [elementIds, elements, setElementOverride, slideInstance]);
 
   const handleDrag = useCallback(
-    (dimensions: Dimensions) => {
-      if (!startDimensions) {
-        return;
-      }
-
-      if (element?.type === 'shape') {
-        setElementOverride([
-          {
-            elementId,
-            elementOverride: {
-              height: dimensions.height,
-              width: dimensions.width,
-              position: { x: dimensions.x, y: dimensions.y },
-            },
-          },
-        ]);
-      } else if (
-        dimensions.elementKind === 'line' ||
-        dimensions.elementKind === 'polyline'
-      ) {
-        setElementOverride([
-          {
-            elementId,
-            elementOverride: {
-              position: { x: dimensions.x, y: dimensions.y },
-              points: dimensions.points,
-            },
-          },
-        ]);
-      }
-    },
-    [element, elementId, setElementOverride, startDimensions],
+    (elementOverrideUpdates: ElementOverrideUpdate[]) =>
+      setElementOverride(elementOverrideUpdates),
+    [setElementOverride],
   );
 
-  if (!element) {
+  if (activeElements.length === 0) {
     return null;
   }
 
-  if (element.type === 'path' && element.kind === 'line') {
-    const renderProperties = getRenderProperties(element);
+  if (
```

Comment:
Is this the case where only one line is selected?
This could use a comment to quickly understand it.